boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -33,7 +33,7 @@ DBG_DEFAULT_CHANNEL(MEMORY);
 #define ULONGLONG_ALIGN_UP_BY(size, align) \
     (ULONGLONG_ALIGN_DOWN_BY(((ULONGLONG)(size) + align - 1), align))
 
-#define MAX_BIOS_DESCRIPTORS 80
+#define MAX_BIOS_DESCRIPTORS (ULONG) 80
```

Comment:
This needs parentheses. You can just use an UL suffix instead though.

Reply:
Agreed. Done.